### DISCUSSION TITLE
About basicVSR++

### DISCUSSION THREAD

**Discussion by 2793145003** [open-mmlab/mmediting#1215] [How-to] [answered]
我使用
```
python demo/restoration_video_demo.py \
    configs/restorers/basicvsr_plusplus/basicvsr_plusplus_c64n7_8x1_600k_reds4.py \
    basicvsr_plusplus_c64n7_8x1_600k_reds4_20210217-db622b2f.pth \
    data/my_data ./outputs
```
试图对小兵张嘎的黑白电影片段进行超分，得到结果
![image](https://user-images.githubusercontent.com/24595900/148163322-8013d9e1-db57-4f34-b27b-1dd8603f4524.png)
因为结果很怪，所以使用钢铁侠第一部的电影，在相同的config的基础上重新训练，得到结果
![image](https://user-images.githubusercontent.com/24595900/148163534-0a60b357-446c-4b57-9132-1d3e3c2a8a63.png)
虽然好了很多，但明显裤子和手的部分也有奇怪的格子……

这些格子是怎么出现的？是训练数据的问题还是模型本身的问题？有没有什么好的建议？

**Comment by ckkelvinchan** [comment, marked as answer]
這某程度上是和退化相關的, 當我們訓練時使用的退化和測試不同的時候, 是有可能會產生這樣的情況
